test/unittests/test_tts.py

```diff
@@ -114,23 +120,23 @@ def test_format_speak_tags_with_speech_no_tags(self):
         self.assertEqual(tagged_with_exclusion, valid_output)
 
     def test_playback_thread(self):
-        from ovos_plugin_manager.templates.tts import PlaybackThread
```

Comment:
Were these imports intentionally removed? They do ensure imports are valid for some minimal testing

Reply:
these classes are part of ovos-audio and should be tested there not here

Reply:
https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/256